### ISSUE TITLE
Install packages on vagrant up

### ISSUE THREAD

**Issue by clubdeuce** [wplib/wplib-box#240]

**Comment by clubdeuce** [comment]
@mikeschinkel Shall we punt this to a later version?

**Comment by mikeschinkel** [comment]
@clubdeuce Yes, we should postpone.  The next release should be all about stabilizing and restoring the ability to work with the use-cases that were workable in `0.11.0`.  Those include but are not limited to wp-cli, modifying config files for Nginx or Apache, etc.
